package-name-disputes.md

```diff
@@ -0,0 +1,131 @@
+   of the critically acclaimed and widely-marketed `foo` JavaScript
+   toolkit framework.  They publish it to npm as `foojs`, but people
+   are routinely confused when `npm install foo` is some different
+   thing.
+4. Yusuf writes a parser for the widely-known `foo` file format, because
+   he needs it for work.  Then, he gets a new job, and never updates
+   the prototype.  Later on, Alice writes a much more complete `foo`
+   parser, but can't publish, because Yusuf's `foo` is in the way.
+
+The validity of Alice's claim in each situation can be debated.
+However, Alice's appropriate course of action in each case is the same.
+
+1. `npm owner ls foo`.  This will tell Alice the email address of the
+   owner (Yusuf).
+2. Alice emails Yusuf:
+    - explaining the terms of her claim to the package name
+    - what she would like to do with the module name
+    - CC's npm support staff <support@npmjs.com>
+    - if appropriate, suggests that Yusuf can run `npm owner add
+      alice foo` to add Alice as an owner of the `foo` package.
+3. After a reasonable amount of time, if Yusuf has not responded, or if
+   Yusuf and Alice can't come to any sort of resolution, support
+   <support@npmjs.com> will follow up to help moderate and move the
+   conversation. ("Reasonable" is usually at least 4 weeks.)
+
+## Assumptions and Values 
+
+These are the base assumptions and values by which naming disputes will
+be evaluated by the npm team:
+
+- A package with a given name should be the package that the most
+  people expect it to be, this means:
+
+    - All things being equal, a package should remain the same package
+    - A popular package name will always point to the same package
+    - A popular trademark will point to an associated package
+
+## Power to the People
+
+In almost every case so far, the parties involved have been able to
+reach an amicable resolution without any major intervention.  Most
+people really do want to be reasonable, and are probably not even
+aware that they're in your way.
+
+Module ecosystems are most vibrant and powerful when they are as
+self-directed as possible.  If an admin one day deletes something you
+had worked on, then that is going to make most people quite upset,
+regardless of the justification.  When humans solve their problems by
+talking to other humans with respect, everyone has the chance to end
+up feeling good about the interaction.
+
+## Exceptions
+
+Some things are not allowed, and will be removed without discussion if
+they are brought to the attention of the npm registry admins,
+including but not limited to:
+
+1. Malware (that is, a package designed to exploit or harm the machine
+   on which it is installed).
+2. Violations of copyright or licenses (for example, cloning an
+   MIT-licensed program, and then removing or changing the copyright
+   and license statement).
+3. Illegal content.
+4. "Squatting" on a package name that you *plan* to use, but aren't
```

Comment:
4b. "Typosquatting", i.e. a *deliberately* confusing package meant to pick up attempts to install another package. If two very different packages have similar names that's okay.